protected.go

```diff
@@ -180,7 +190,7 @@ func (p *Protector) dial(op ops.Op, network, addr string, timeout time.Duration)
 	}
 	copy(conn.ip[:], ip.To4())
 
-	socketFd, err := syscall.Socket(syscall.AF_INET, syscall.SOCK_STREAM, 0)
+	socketFd, err := syscall.Socket(syscall.AF_INET, socketType, 0)
```

Comment:
Maybe that's because few Android phones support IPv6 well (http://www.techrepublic.com/article/androids-lack-of-dhcpv6-support-poses-security-and-ipv6-deployment-issues/) ? Plus most home wifi / operator network are still IPv4